wagtailmedia/widgets.py

```diff
@@ -44,3 +67,23 @@ class Media:
             "wagtailmedia/js/media-chooser-modal.js",
             "wagtailmedia/js/media-chooser.js",
         ]
+
+
+if WAGTAIL_VERSION > (2, 13):
```

Comment:
when the imports switch to a try/except there's no need for the version check.

note that as it stands `WAGTAIL_VERSION > (2, 13)` doesn't actually work with 2.13..